nilearn/maskers/surface_labels_masker.py

```diff
@@ -572,28 +572,29 @@ def _create_figure_for_report(self):
             img = mean_img(img)
             vmin, vmax = img.data._get_min_max()
 
-        # TODO: possibly allow to generate a report with other views
-        views = ["lateral", "medial"]
-        hemispheres = ["left", "right"]
+        hem_view = {
```

Comment:
```suggestion
        hemi_view = {
``` (nitpick, but I find this more readable)